From 70835aabbb1c7de75f36c1948a5f0cf734ebf6ee Mon Sep 17 00:00:00 2001
From: Will Tisdale <willtisdale@gmail.com>
Date: Sun, 9 Sep 2012 00:13:21 +0100
Subject: [PATCH 2/3] grouper: PowerHAL: Cleanup remove 700MHz clamp whilst
 screen off (leftover from tuna) remove any interactive
 governor related code

---
 power/power.c |    2 --
 1 file changed, 2 deletions(-)

diff --git a/power/power.c b/power/power.c
index bc6ec64..255d694 100644
--- a/power/power.c
+++ b/power/power.c
@@ -90,8 +90,6 @@ static int boostpulse_open(struct grouper_power_module *grouper)
 
 static void grouper_power_set_interactive(struct power_module *module, int on)
 {
-    sysfs_write("/sys/devices/system/cpu/cpu0/cpufreq/scaling_max_freq",
-                on ? "1500000" : "700000");
     sysfs_write(SAMPLING_RATE_ONDEMAND,
                 on ? SAMPLING_RATE_SCREEN_ON : SAMPLING_RATE_SCREEN_OFF);
 }
-- 
1.7.9.5

